openbas-api/src/main/java/io/openbas/service/InjectExpectationService.java

```diff
@@ -110,7 +110,7 @@ public InjectExpectation updateInjectExpectation(
       injectExpectation.setScore(input.getScore());
     } else {
       if (input.getScore() > injectExpectation.getScore()
-          || MANUAL.equals(injectExpectation.getType())) {
+          || List.of(MANUAL, CHALLENGE, ARTICLE).contains(injectExpectation.getType())) {
```

Comment:
nitpick: Could we build this List as a constant maybe, as it's used at least twice?